tests/test_vcs_git/test_vcs_git.py

```diff
@@ -0,0 +1,42 @@
+from os.path import dirname, join
+from shutil import copyfile
+
+from dxr.testing import DxrInstanceTestCase
+
+from nose import SkipTest
+from nose.tools import ok_
+
+
+class GitTests(DxrInstanceTestCase):
+    """Tests that the diff links for files go somewhere helpful"""
```

Comment:
One thing that sometimes differs on CI is hash order. Dicts can differ in their order of iteration from Python version to Python version. Is there anything here that depends on order?